packages/spec/src/index.ts

```diff
@@ -1,6 +1,106 @@
-import type * as v1 from "./v1.ts";
+export declare namespace v1 {
+  /**
+   * The Standard Schema interface.
+   */
+  export interface StandardSchema<Input = unknown, Output = Input> {
+    /**
+     * The Standard Schema properties.
+     */
+    readonly "~standard": StandardSchemaProps<Input, Output>;
+  }
 
-export type StandardSchema = v1.StandardSchema;
+  /**
+   * The Standard Schema properties interface.
+   */
+  export interface StandardSchemaProps<Input = unknown, Output = Input> {
+    /**
+     * The version number of the standard.
+     */
+    readonly version: number;
+    /**
+     * The vendor name of the schema library.
+     */
+    readonly vendor: string;
+    /**
+     * Validates unknown input values.
+     */
+    readonly validate: (
+      value: unknown,
+    ) => StandardOutput<Output> | Promise<StandardOutput<Output>>;
```

Comment:
I think we should rename all the `StandardOutput` to `StandardResult` to account for a future codec spec. 

Otherwise we'll end up with some weirdness down the line e.g. `StandardOutput<Input>`. From my proposal on Discord:

```ts
export interface StandardCodec<Input = unknown, Output = Input>
  extends StandardSchema<Input, Output> {
  readonly "~decode": (
    input: Input,
    options?: undefined,
  ) => StandardOutput<Output> | Promise<StandardOutput<Output>>;
  readonly "~encode": (
    input: Output,
    options?: undefined,
  ) => StandardOutput<Input> | Promise<StandardOutput<Input>>;
}
```

Reply:
Done